### PR TITLE
Export the kubeconfig path during executing the following steps.

### PR DESCRIPTION
**Pull request by dashanji**
As titled.

### CODE REVIEW

File: internal/components/setup/kind.go [hunk 1]
```diff
@@ -106,6 +106,13 @@ func KindSetup(e2eConfig *config.E2EConfig) error {
 		if err := createKindCluster(kindConfigPath, e2eConfig); err != nil {
 			return err
 		}
+	} else {
+		// export the kubeconfig path for command line
+		err := os.Setenv("KUBECONFIG", kubeConfigPath)
+		if err != nil {
+			return fmt.Errorf("could not export kubeconfig file path, %v", err)
+		}
+		logger.Log.Infof("export KUBECONFIG=%s", kubeConfigPath)
```
Comment (fgksgf) [inline internal/components/setup/kind.go:115]: After the test, shall we unset the env var?

Comment (fgksgf) [reply internal/components/setup/kind.go:115]: What do you think? @kezhenxu94

Comment (kezhenxu94) [reply internal/components/setup/kind.go:115]: > After the test, shall we unset the env var?

This is to make it convenient running other commands in terminal after e2e command right? If so we might want to unset the env var in `cleanup` command, if possible, we might even want to revert `KUBECONFIG` to the original value before we change it.

Comment (kezhenxu94) [reply internal/components/setup/kind.go:115]: OK. I thought you want to persist the env var after the e2e exits. LGTM

**Pull request merged**